Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -3061,6 +3061,8 @@ extension BrowserViewController: PreferencesObserver {
             setupTabs()
             updateTabsBarVisibility()
             updateApplicationShortcuts()
+            tabManager.removeAll()
```

Comment:
We need to reset config & remove all tabs even if the user is turning the feature _off_?

Reply:
@kylehickinson when you turn PBO off, it automatically switches you to Normal mode. This was the behaviour from long before.
Only thing I did was add a call to removeAll and resetConfiguration.. which I guess isn't necessary. I just tested and it seems it makes no difference.